kpatch/kpatch

```diff
@@ -163,7 +163,16 @@ load_module () {
 	fi
 
 	echo "loading patch module: $1"
-	insmod "$1" "$2"
+	for i in `seq 5`; do
+		if [[ $i != 1 ]]; then
+			echo "retrying..."
+			sleep 2
+		fi
+		insmod "$1" "$2" && break
```

Comment:
Do we still need to call `insmod` with `$2`? Looks like nothing in the script sets it anymore. It used to be used to pass the `replace` param, but IIRC the `kpatch replace` option was deprecated in commit 8e8de4718d8